### PR TITLE
First tutorial: governance with Prolog

### PR DESCRIPTION
**Pull request by antho31**
* Overview with link to Prolog tutorial 
* Complete first tutorial, Prolog & `okp4d` commands
* `.markdownlint.yaml` updated to enable ordered lists with code blocks

### CODE REVIEW

**Review by ccamel — CHANGES_REQUESTED** [review]
Excellent tutorial ! I have a few minor comments that you can consider at your convenience.

File: docs/tutorials/prolog-1.mdx [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+--label "rules-tutorial-ex-$(date +%s)" \
+--no-admin \
+--chain-id okp4-nemeton-1 \
+--gas 1000000 \
+--node https://api.testnet.okp4.network:443/rpc \
+"{\"program\":\"$(cat my_knowledgebase.pl | base64)\", \"storage_address\": \"okp416zrd20eqwx0nv05pu45na6jq345r6ggqrgx2sh79ln5z8vqvztgsqsfr4p\"}"
+```
+
+Replace the `--from` value by your wallet name. You can also change the `—label` name as you want but note it should be unique.
+You can replace `—no admin` with `—admin YOUR_OKP4_ADDR` if you want to be able to “break” (to remove the stored program).
```
Comment (bdeneux) [inline docs/tutorials/prolog-1.mdx:291]:
```suggestion
You can replace `—no-admin` with `—admin YOUR_OKP4_ADDR` if you want to be able to “break” (to remove the stored program).
```

**Review by bdeneux — CHANGES_REQUESTED** [review]
Wonderful tutorial ! 👏 Nice ! I just notice one mistake :D

File: docs/tutorials/prolog-1.mdx (hunk 1 above)
Comment (antho31) [reply docs/tutorials/prolog-1.mdx]: Thanks for your feedback, and wow good point :) !

File: docs/tutorials/prolog-1.mdx [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+--label "rules-tutorial-ex-$(date +%s)" \
+--no-admin \
+--chain-id okp4-nemeton-1 \
+--gas 1000000 \
+--node https://api.testnet.okp4.network:443/rpc \
+"{\"program\":\"$(cat my_knowledgebase.pl | base64)\", \"storage_address\": \"okp416zrd20eqwx0nv05pu45na6jq345r6ggqrgx2sh79ln5z8vqvztgsqsfr4p\"}"
+```
+
+Replace the `--from` value by your wallet name. You can also change the `—label` name as you want but note it should be unique.
+You can replace `—no-admin` with `—admin YOUR_OKP4_ADDR` if you want to be able to “break” (to remove the stored program).
```
Comment (amimart) [inline docs/tutorials/prolog-1.mdx:291]:
```suggestion
Replace the `--from` value by your wallet name. You can also change the `--label` name as you want but note it should be unique.
You can replace `--no-admin` with `--admin YOUR_OKP4_ADDR` if you want to be able to “break” (to remove the stored program).
```

**Review by amimart — CHANGES_REQUESTED** [review]
That's really great! 💪

**Review by bdeneux — APPROVED** [review]

**Review by MdechampG — APPROVED** [review]

**Pull request merged**